core/perf_test/CMakeLists.txt

```diff
@@ -82,7 +78,7 @@ FUNCTION(KOKKOS_ADD_BENCHMARK NAME)
     )
   ENDIF()
 
-  SET(BENCHMARK_NAME ${PACKAGE_NAME}_${NAME})
+  SET(BENCHMARK_NAME Kokkos_${NAME})
```

Comment:
Why is hardcoded name preferred?

Reply:
I didn't see much of a reason to keep a variable that always has the same value for it.